content/docs/user-guide/components/reference/terrain/surface-material-list.md

```diff
@@ -15,13 +18,18 @@ Select the surface type tag using the **SurfaceTag** drop-down menu, then assign
 
 [Terrain Gem](/docs/user-guide/gems/reference/environment/terrain)
 
+## Dependencies
+
+[Axis-Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape)
+
 ## Properties
 
 ![Terrain Surface Materials List component properties](/images/user-guide/components/reference/terrain/terrain-surface-materials-list-component.png)
 
 | Property | Description | Values | Default |
 |-|-|-|-|
-| **Gradient to Material Mappings** | An array of [surface tags](/docs/user-guide/gems/reference/environment/surface-data) and material assets to map together. |  |  |
+| **Default Material** | The default material to fall back to where no other material surface mappings exist. | Material Asset | None |
```

Comment:
Either here or above we should mention that the default material doesn't blend with other materials, because it doesn't have a surface weight. So for example, if you have a default material of grass, and a material mapping with dirt, even if the dirt has a weight of 0.01, you will see 100% dirt and no grass.

The primary intended uses of the default material are either to use an "error" material to see every place that a detail material hasn't been mapped, or to just quickly cover an entire terrain surface with a single material without setting up more complicated mappings.

Reply:
We may be able to make the detail material more useful in the future, but as @mbalfour-amzn said, it's mostly a placeholder for now.

Reply:
Good call!